lib/src/components/NxBinaryDonutChart/__tests__/NxDonutChart.test.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+import NxBinaryDonutChart from '../NxBinaryDonutChart';
+
+describe('NxBinaryDonutChart', function() {
+  it('renders an svg with the expected properties', function() {
+    const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, { percent: 90 });
+
+    expect(getShallowComponent()).toMatchSelector('svg');
+    expect(getShallowComponent()).toHaveProp('viewBox', '-100 -100 200 200');
+    expect(getShallowComponent()).toHaveClassName('.nx-binary-donut-chart');
+  });
+
+  it('renders only the background circle with the expected properties at zero percent', function() {
+    const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, { percent: 0 });
+    const circles = getShallowComponent().find('circle');
+    const paths = getShallowComponent().find('path');
+
+    expect(circles.length).toBe(1);
+    expect(circles.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
+    expect(circles.at(0)).toHaveProp('strokeWidth', 50);
+    expect(circles.at(0)).toHaveProp('r', 75);
+    expect(paths.length).toBe(0);
+  });
+
+  it('renders only a background circle with the expected properties when percent is 0', function() {
```

Comment:
Isn't this the same as the previous test?

Reply:
> Isn't this the same as the previous test?

I modified the test for another percentage ec3dc386383acec3217b81fca65f5651fd594ce8

Reply:
> I modified the test for another percentage ec3dc38

"it renders only a background circle with the expected properties when percent is 50" is not desired to be a true statement, nor is it what that test actually tests.

Reply:
> > I modified the test for another percentage [ec3dc38](https://github.com/sonatype/sonatype-react-shared-components/commit/ec3dc386383acec3217b81fca65f5651fd594ce8)
> 
> "it renders only a background circle with the expected properties when percent is 50" is not desired to be a true statement, nor is it what that test actually tests.

I removed this test since I realized 50 was already being tested, I added two more with different values